entities/entities/spawned_shipment/init.lua

```diff
@@ -92,8 +92,8 @@ function ENT:Use(activator, caller)
 
     local canUse, reason = hook.Call("canDarkRPUse", nil, activator, self, caller)
     if canUse == false then
-        if reason then DarkRP.notify(activator, 1, 4, reason) end
-        return
+      if reason then DarkRP.notify(activator, 1, 4, reason) end
+      return
```

Comment:
I think there is an indentation problem here.

Reply:
This code is not modified besides the indentation. I think this change can be undone.